Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist

```diff
@@ -1340,7 +1572,7 @@ changing “passwordID” to the correct element ID. If the value you typed into
 		</dict>
 		<dict>
 			<key>pfm_description</key>
-			<string>Name for Shares menu item. Default: "Shares".</string>
+			<string>Name for Shares menu item. Default: &quot;Shares&quot;.</string>
```

Comment:
Is this an intentional change, it displays fine as is in both ProfileCreator and Imazing Profile Editor.